### PR TITLE
CPM-132: User export via flat files

### PR DESCRIPTION
**Pull request by mmetayer**
**Description (for Contributor and Core Developer)**

User export. The avatar image is not exported yet, done in #13926 

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | OK
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | OK
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/back/UserManagement/Specification/Component/Normalizer/Standard/UserNormalizerSpec.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        $avatar->getKey()->willReturn('a/b/c/avatar.png');
+
+        $user->getUsername()->willReturn('johndoe');
+        $user->isEnabled()->willReturn(true);
+        $user->getNamePrefix()->willReturn(null);
+        $user->getFirstName()->willReturn('John');
+        $user->getMiddleName()->willReturn(null);
+        $user->getLastName()->willReturn('Doe');
+        $user->getNameSuffix()->willReturn(null);
+        $user->getPhone()->willReturn('+330000000');
```
Comment (pierallard) [inline tests/back/UserManagement/Specification/Component/Normalizer/Standard/UserNormalizerSpec.php:59]: its phone number is amazing

**Review by pierallard — APPROVED** [review]

File: src/Akeneo/UserManagement/Bundle/Resources/config/jobs.yml [hunk 2]
```diff
@@ -49,6 +48,30 @@ services:
             - { name: akeneo_batch.job, connector: '%pim_connector.connector_name.xlsx%', type: '%pim_connector.job.export_type%' }
 
     # Import jobs
+    pim_connector.job.csv_user_export:
```
Comment (nmarniesse) [inline src/Akeneo/UserManagement/Bundle/Resources/config/jobs.yml:51]: should be before `# Import jobs` line

Comment (nmarniesse) [inline src/Akeneo/UserManagement/Bundle/Resources/config/jobs.yml:51]: should be before `# Import jobs` line

**Review by nmarniesse — APPROVED** [review]

File: src/Akeneo/UserManagement/Bundle/Resources/config/jobs.yml (hunk 2 above)
Comment (mmetayer) [reply src/Akeneo/UserManagement/Bundle/Resources/config/jobs.yml:51]: I thought I had removed this line actually, I must have messed up with my rebase :smile:

File: tests/back/UserManagement/Specification/Component/Normalizer/Standard/UserNormalizerSpec.php (hunk 1 above)
Comment (mmetayer) [reply tests/back/UserManagement/Specification/Component/Normalizer/Standard/UserNormalizerSpec.php:59]: isn't it? :slightly_smiling_face:

**Pull request merged**